qiskit/extensions/quantum_initializer/initializer.py

```diff
@@ -363,7 +363,83 @@ def validate_parameter(self, parameter):
 
 
 def initialize(self, params, qubits=None):
-    """Apply initialize to circuit."""
+    """Qubit initializalition is done by appending instructions to the quantum circuit (by
+    calling Initialize(params)) and the qubits we wish to iniatilize. Note that the
```

Comment:
```suggestion
    calling Initialize(params)) and the qubits we wish to initialize. Note that the
```

Reply:
Thanks for catching that, all your suggestions have been addressed in d483846.